test.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
```

Comment:
I guess we need this also in deploy-goerli.sh and deploy-mainnet.sh right?

Reply:
done.